std/experimental/allocator/mallocator.d

```diff
@@ -68,16 +68,14 @@ struct Mallocator
 }
 
 ///
-@nogc nothrow
-@system unittest
+@nogc @system nothrow unittest
 {
     auto buffer = Mallocator.instance.allocate(1024 * 1024 * 4);
```

Comment:
I wonder why these lines appear green? Is it because they weren't covered and now they are?

Reply:
I have not the CodeCov plugin enabled and i see it green too, so it's the standard diff color.